.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+env:
+  DEVELOPER_DIR: /Applications/Xcode_11.2.app/Contents/Developer
+  WORKSPACE: ReactiveLists.xcworkspace
+  SCHEME: ReactiveLists
+  IOS_SDK: iphonesimulator13.2
+
+jobs:
+  iOS:
+    name: iOS Test
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        destination: ["OS=13.2,name=iPhone 11"]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Switch to Xcode 11
+        run: sudo xcode-select --switch /Applications/Xcode_11.1.app
```

Comment:
Is this step necessary if we've specified a specific `IOS_SDK` above?

Reply:
Oh true don't need this because of `DEVELOPER_DIR`